README.md

```diff
@@ -1,77 +1,150 @@
-# Role Name
+# mssql
+
 ![CI Testing](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This role installs, configures, and starts Microsoft SQL Server (MSSQL).
+The role currently uses MSSQL version 2019 only.
```

Comment:
Just out of curiousity..., once mssql version 202X is released and determined to support in the linux-system-roles, how are we include it? One version at a time and replace entirely? Or could the two versions coexist?

Reply:
Don't know yet - presumably Microsoft will provide some sort of upgrade path

Reply:
Ok. The reason why I asked was the role contains hardcoded "2019" in it (of course, not that many, though). Once migrated to 202X, we are planing to rewrite the role somehow?

Reply:
Yes, we are planning to change the role to support 2017 in the near future, then probably 202X when available.

Reply:
I wanted to clarify that the role really only does 2019 MSSQL now. I am going to add support for 2017 by Q2 or Q3 after I add `advanced` features to the role that include HA. And with each new version added I am going to explicitly state everywhere what versions the role supports.

Reply:
Shoudn't you treat different versions of mssql similarly to providers in some other roles?

Reply:
> Shoudn't you treat different versions of mssql similarly to providers in some other roles?

That's one way to do it.  The mssql is not a system role - that is, it isn't managing a linux system component which could have different providers depending on the OS version - so I'm not sure if provider is the right concept here.  We'll have to figure out the best approach once we start implementing support for 2017.